index.html

```diff
@@ -5007,13 +4989,281 @@ <h3 id="looping-AudioBufferSourceNode">
+              for interpolation later.
+            </li>
+            <li>For each block of audio to be rendered:
+              <ol>
+                <li>Let <code>currentTime</code> be the current time of the <a>
+                  AudioContext</a>.
+                </li>
+                <li>Let <code>dt</code> be 1 / (_context sample rate_).
+                </li>
+                <li>Let <code>index</code> be 0.
+                </li>
+                <li>Let <code>computedPlaybackRate</code> be <code>playbackRate
+                * pow(2, detune / 1200) * (bufferRate / contextRate)</code> .
+                This combines the two k-rate parameters affecting playback rate
+                into a single computed quantity, and also corrects for any
+                difference in sample rate between the buffer and the
+                <a>AudioContext</a>.
+                </li>
+                <li>
+                  <em>Optimize the buffer</em> prior to rendering the block, if
+                  desired.
+                </li>
+                <li>While <code>index</code> is less than the length of the
+                audio block to be rendered:
+                  <ol>
+                    <li>If <code>currentTime &lt; start</code> or
+                    <code>currentTime &gt;= stop</code>, emit silence for the
+                    output frame at <code>index</code>.
+                    </li>
+                    <li>Else,
+                      <ol>
+                        <li>If <code>bufferTime</code> &lt; 0, set
+                        <code>bufferTime</code> to <code>offset + ((currentTime
+                        - start) * computedPlaybackRate)</code>. This is the
+                        playhead position for the first output sample frame.
+                        </li>
+                        <li>Let <code>effectiveBufferTime</code> equal
+                        <code>bufferTime</code>.
+                        </li>
+                        <li>While <code>loop</code> is true and
+                        <code>effectiveBufferTime</code> &gt;=
+                        <code>actualLoopEnd</code>, map the playhead position
+                        to the correct point within the loop body accounting
+                        for loop iterations:
+                          <ol>
+                            <li>Let <code>effectiveBufferTime</code> equal
+                            <code>effectiveBufferTime - (actualLoopEnd -
+                            actualLoopStart)</code>.
+                            </li>
+                          </ol>
+                        </li>
+                        <li>Emit the result of
+                        <code>playbackSignal(effectiveBufferTime)</code> as the
+                        output frame at <code>index</code>.
+                        </li>
+                      </ol>
+                    </li>
+                    <li>Increase <code>bufferTime</code> by <code>dt *
```

Comment:
I think this is at the wrong level and should be in the Else clause.  Otherwise bufferTime gets incremented even if currentTime < start.